plugins/worker/server/__init__.py

```diff
@@ -94,6 +117,34 @@ def schedule(event):
         })
 
 
+def cancel(event):
+    """
+    This is bound to the "jobs.cancel" event, and will be triggered any time
+    a job is canceled. This handler will process any job that has the
+    handler field set to "worker_handler".
+    """
+    job = event.info
+    if job['handler'] == 'worker_handler':
```

Comment:
```worker_handler``` is the handler that manages traditional ```app.send_message(...)``` task production.  Please also white list ```"celery_handler"``` which is the handler type we've assigned to traditional celery based task distribution (e.g. with ```my_task.delay(...)``` and ```my_task.apply_async(...)```)